tests/controllers/event.test.js

```diff
@@ -267,19 +267,6 @@ tape('DELETE /events/:id with invalid id', t => {
     })
 })
 
-tape('DELETE /users/:id with id of something not in the database', t => {
```

Comment:
please could you reinstate this, I believe it will cause a merge conflict with my branch otherwise.

Reply:
@mattlub It fails the tests though and shouldn't be in there? If I reinstate it it will need to be changed anyway to pass the tests - makes more sense to fix the merge conflict if/when it happens?

Reply:
ah ok